composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": "^7.2",
         "doctrine/annotations": "^1.2",
-        "doctrine/mongodb-odm": "^2.0.0-beta1",
+        "doctrine/mongodb-odm": "dev-master",
```

Comment:
Should this be changed to 2.0.0-RC2 (following its release, of course)?

EDIT: Just saw the "REMOVE:" commit. Makes sense.